lib/cli/src/commands/create_exe.rs

```diff
@@ -562,26 +564,71 @@ impl CreateExe {
             include_dir.pop();
             include_dir.push("include");
 
+            let mut default_include_path = None;
+
+            #[cfg(target_os = "macos")]
+            {
+                if let Ok(output) = Command::new("xcrun").arg("--show-sdk-path").output() {
+                    default_include_path =
+                        Some(String::from_utf8_lossy(&output.stdout).to_string());
+                }
+            }
+
+            #[cfg(target_os = "linux")]
+            {
+                default_include_path = Some("/usr/include".to_string());
+            }
+
+            #[cfg(target_os = "windows")]
+            {
+                // TODO: discover include path?
+                default_include_path = Some(
+                    "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.17134.0\\ucrt"
+                        .to_string(),
+                );
+            }
```

Comment:
Why would we need this if we are cross-compiling?

Reply:
I was not aware that `-isystem` exists, so yesterday I tried fixing the issue "manually". The issue was that `#include <stdio.h>` for example wasn't found because the zig cc command doesn't include the default system headers.

I will remove this block and replace it with `-isystem`, hopefully that will work.